network-api/networkapi/wagtailpages/models.py

```diff
@@ -1276,3 +1277,27 @@ def serve(self, request):
         # Note that due to how this page type works, there is no
         # associated template file in the wagtailpages directory.
         return HttpResponseRedirect(self.URL)
+
+
+class SpecialTemplatePage(FoundationMetadataPageMixin, Page):
+    ALLOWED_TEMPLATES = [
+        ('unspecified', '-'),
+        ('wagtailpages/one-off-pages/youtube.html', 'Youtube Campaign'),
+    ]
+
+    template = models.CharField(
+        choices=ALLOWED_TEMPLATES,
+        default='unspecified',
+        max_length=500,
+        help_text='The template to be used to render this one-off page',
+        blank=True,
+    )
+
+    textfields = StreamField([
+        ('textblock', customblocks.TRKeyValueBlock()),
+    ])
```

Comment:
by using a streamfield, localization is "for this one streamfield", no matter how many key/value pairs that consists of. Thankfully the "copy from [EN] [...] ..." means that starting a localization will bootstrap off of preexisting fields, but I don't know if this is what we want to run with.